src/mame/video/gaplus.cpp

```diff
@@ -247,7 +249,7 @@ void gaplus_base_state::starfield_render(bitmap_ind16 &bitmap)
 		int x = m_stars[i].x;
 		int y = m_stars[i].y;
 
-		/* Some stars in the second tier will flash erratically while changing their movements. */
+		/* Some stars in the second layer will flash erratically when changing their movements */
```

Comment:
Does this happen on real HW as well? If yes please indicate it, if no please consider refactor this comment with a C++ [TODO style](https://help.semmle.com/wiki/display/CCPPOBJ/TODO+comment)

Reply:
This can be seen in the gameplay video on the PCB.
This happens when reverse scrolling after a challenging stage (e.g. PARSEC 4) or scrolling sideways when an elephant is acquired (e.g. PARSEC 16).
(That is, except when scrolling normally.)

https://www.youtube.com/watch?v=_1x5Oid3uPg
3:35- 13:12-

https://www.youtube.com/watch?v=vrmZAUJYXnI
3:14- 12:40-

Reply:
I'll add to the comments that the pattern of flashing is based on guess from video.
Thank you for review.